setup.sh

```diff
@@ -19,6 +19,12 @@ scram project ${CMSSWVER}
 cd ${CMSSWVER}/src/
 # cmsenv
 eval `scramv1 runtime -sh`
+echo "Needs your CERN username and pass word: NNKit is being cloned from gitlab"
```

Comment:
This is quite unfortunate. It means the setup script can no longer be automated. We should look for ways around this.